armory/utils/export.py

```diff
@@ -64,7 +93,15 @@ def _make_output_dir(self):
             )
         os.mkdir(self.output_dir)
 
-    def _export_images(self, x, x_adv):
+    def _export_images(
+        self, x, x_adv, y=None, y_pred_adv=None, y_pred_clean=None, classes_to_skip=None
+    ):
+
+        plot_boxes = True if y is not None else False
```

Comment:
The logic of `plot_boxes = True if y is not None else False` doesn't seem intuitive. This variable also feels redundant with `plot_bboxes`. Maybe exporting object detection should just have its own function?